### PR TITLE
Add test step in pr-check action

### PR DESCRIPTION
**Pull request by Bruception**
### Description

Closes #

### CODE REVIEW

File: backend/app.ts [hunk 1]
```diff
@@ -8,6 +8,8 @@ import errorHandlingMiddleware from "./middlewares/error";
 function buildApp(): express.Application {
   const app = express();
 
+  console.log("TEST!");
```
Comment (sondrekje) [inline backend/app.ts:11]: why? 😅

File: .github/workflows/pr-check.yml [hunk 2]
```diff
@@ -89,9 +89,11 @@ jobs:
         if: steps.filter.outputs.any-tsscss == 'true'
         run: npm run lint
 
-      - name: Build backend
+      - name: Build/Test backend
         if: steps.filter.outputs.ts-be == 'true'
-        run: npm run pr-check-build-be
+        run: |
+          npm run pr-check-build-be
+          npm run test-backend
```
Comment (sondrekje) [inline .github/workflows/pr-check.yml:96]: would a coverage report be nice to have?

File: backend/app.ts (hunk 1 above)
Comment (Bruception) [reply backend/app.ts:11]: Build step won't run if no changes to backend directory

File: .github/workflows/pr-check.yml (hunk 2 above)
Comment (Bruception) [reply .github/workflows/pr-check.yml:96]: We should do coverage in a separate PR. Have to think about how we want to set that up.

**Review by Miodec — APPROVED** [review]
Looks good, thanks.

**Pull request merged**